tests/test_download.py

```diff
@@ -761,37 +761,38 @@ def mock_api_call(self, operation_name, api_params):
 
 
 @pytest.mark.django_db
-def test_missingsymbol_model_counts():
-    # The .total_count won't change just because you manually
-    # create, edit, or delete MissingSymbol instances. The increments are
-    # done explicitly in the code where upserts are made.
-    assert MissingSymbol.total_count() == 0
+def test_cleanse_missingsymbol_delete_records():
```

Comment:
This is an unfortunate diff. I removed the `test_missingsymbol_model_counts` function completely. I added a totally different and unrelated `test_cleanse_missingsymbol_delete_records` test.